app/konnect/dedicated-cloud-gateways.md

```diff
@@ -0,0 +1,31 @@
+---
+title: About Cloud Gateways
+content_type: concept
+---
+
+Dedicated cloud gateways are a type of API gateway in {{site.konnect_short_name}} that are completely managed by the cloud provider of your choice. You maintain control over the size and location of the gateway infrastructure, while Kong oversees the management of each instance and the entire cluster for you.
+
+You can manage your dedicated cloud gateways alongside all your other gateways in [Gateway Manager](https://cloud.konghq.com/gateway-manager/).
+
+![cloud gateway dashboard](/assets/images/products/konnect/gateway-manager/konnect-control-plane-cloud-gateway.png)
+> _**Figure 1:** Example of the dedicated cloud gateway dashboard in Gateway Manager. The dashboard displays the total traffic, error rate, and P99 latency. It also displays the top five gateway services and routes by traffic, as well as the plugins and consumers associated with the gateway._
+
+Dedicated cloud gateways have the following benefits over a typical hybrid gateway:
```

Comment:
Can you clarify this for me: can a control plane have a mix of dedicated and regular gateway data plane nodes? Or is each control plane limited to one node type? That might help clear up some confusion. By using the term "hybrid", this sounds like we're comparing a full gateway setup (control plane and data plane), where I thought that CGWs were data planes only.

"Typical" also seems like the wrong word to use here. That might make sense in a brief or something, but I imagine we don't want to position the non-CGW type of nodes as typical in perpetuity.